### PR TITLE
make the bdd with dbmigrate

### PR DESCRIPTION
**Pull request by seb879**
Creating the database and initialisation of the migration

### CODE REVIEW

File: migrations/20190602121612-create-table-point-of-interest.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db) {
+  return db.createTable('pointofinterest', {
```
Comment (erabin05) [inline migrations/20190602121612-create-table-point-of-interest.js]: camelCase or other naming convention for 'pointofinterest'

**Review by erabin05 — CHANGES_REQUESTED** [review]
naming convention for table

File: migrations/20190602121612-create-table-point-of-interest.js (hunk 1 above)
Comment (chandelieraxel) [reply migrations/20190602121612-create-table-point-of-interest.js]: Done

**Review by erabin05 — APPROVED** [review]
Ok

**Pull request merged**